### ISSUE TITLE
Issue Loading the Pack just instantly crashes

### ISSUE THREAD

**Issue by Volutus43** [FTBTeam/FTB-Modpack-Issues#6425]
### Modpack

FTB Evolution

### Modpack version

1.4.1

### Launcher

FTB App

### Has the pack been modified

No

### Describe the bug

So I was playing the Evolution fine on version 1.3.0 with no problems and I updated to the latest version and now the pack will not even load to the main page and it doesn't provide a crash report, only the log. It feels like something to maybe do with neoforge? But I honestly don't know. I have even tried clean reinstall and deleting the instance folder but the same issue of non loading continues. I also even tried to update my java to version 23 but same issue so I went back to java 21.

### Steps to reproduce

1. Start the pack from version 1.3.0
2. Updated the pack to 1.4.1
3. Pack then does not load and "crashes" but does not provide a report just a log

### Expected behaviour

I expected the pack to load as I have not modified anything but it doesn't.

### Additional information

Something that is worth noting is I have been able to run this version before but the only way that it runs is if I spam the pack loading after it just "dies" and it works fine. However, I feel like it should not have to be loaded 10+ times to get it to work and I don't know how to fix it. Other thing to note too is sometimes I have blue screened from loading it too many times and I assume that is from having things occurring in the background but still worrisome to see. Lastly, I am on windows 11 I have 64 GB of ram (only allocated 8 GB for pack) and all other packs work for me besides this one (example Direwolf20 on the same version).

**Comment by Gaz492** [comment]
Would you be able to upload the debug.log for us? I'm not seeing anything problematic in the latest.log you uploaded unfortunately

**Comment by Volutus43** [comment]
[debug.log](https://github.com/user-attachments/files/18149494/debug.log)
Here is that log

**Comment by Gaz492** [comment]
Hmm, the log just shows that the game closed suddenly, there aren't any errors printed out before the log just ends either.

If you install a fresh copy of the modpack on the latest version, does the same issue happen?

**Comment by Volutus43** [comment]
Yes I have done this on my instance and on a clean copy. I even went into my files to make sure it was deleted after deleting the instance.

**Comment by Volutus43** [comment]
I haven't tried 1.5.0 yet if that's what you meant as well just the 1.4.1 version

**Comment by Volutus43** [comment]
So I did for testing reasons load up 1.5.0 and that did work and didn't instantly crash for what its worth.

**Comment by Gaz492** [comment]
Hmm, ill be honest, I'm not sure why 1.5.0 works for you and 1.4.1 didn't, when I sent the message 1.5.0 hadn't been released yet :P but great to hear it works now.

**Comment by Volutus43** [comment]
Yeah its very strange. I'm wondering if it had to do with a mod update maybe or the neoforge but I'm clueless.

**Comment by Gaz492** [comment]
Possibly, ill close this issue as it looks to be resolved, but if it returns, feel free to either reopen this or make a new issue!

**Comment by Volutus43** [comment]
[crash-2024-12-17_15.28.34-fml.txt](https://github.com/user-attachments/files/18171025/crash-2024-12-17_15.28.34-fml.txt)

**Comment by Volutus43** [comment]
Updated crash report now happening on 1.5.0

**Comment by Volutus43** [comment]
I wanted to send this I don't know if it reopened the issue but I tried to this time I got a crash report

**Comment by Gaz492** [comment]
Do you have any antivirus program, it seems that some of the config files for the modpack are being used by something else which is causing the mod to run into an error while the pack starts up

**Comment by Volutus43** [comment]
I don't have an antivirus on unless its windows 11 blocking it?

**Comment by Gaz492** [comment]
Ok, as a test, if you install a new copy of the modpack does the same crash happen?

**Comment by Volutus43** [comment]
I will test now

**Comment by Volutus43** [comment]
so on 1.5.0,  with a fresh install, I didn't get the crash I got the old issue with the log where it just doesn't load

**Comment by Volutus43** [comment]
[latest.log](https://github.com/user-attachments/files/18203070/latest.log)

**Comment by Volutus43** [comment]
[debug.log](https://github.com/user-attachments/files/18203072/debug.log)

**Comment by Volutus43** [comment]
To describe it, it loads to a point (usually the same point every time) then just closes the app without me closing it.

**Comment by Gaz492** [comment]
If you go into the main folder of the modpack (where the saves/config/mods folders is) do you see any files named like hs_err_pid?

**Comment by Volutus43** [comment]
[hs_err_pid14240.log](https://github.com/user-attachments/files/18203129/hs_err_pid14240.log)

**Comment by Volutus43** [comment]
I tried to load 2 times so I have 2 of them if you want both here is the other one.
[hs_err_pid36688.log](https://github.com/user-attachments/files/18203132/hs_err_pid36688.log)

**Comment by Volutus43** [comment]
oh ok thanks for looking out on that I will try removing the mod and see

**Comment by Volutus43** [comment]
So after removing that mod I got further and it loaded

**Comment by Volutus43** [comment]
:D

**Comment by Volutus43** [comment]
I am going to try loading it again to see

**Comment by Volutus43** [comment]
Yeah works perfectly fine now once I removed productivetrees

**Comment by Volutus43** [comment]
wonder what that means now then so I guess is that mod conflicting with something?

**Comment by Volutus43** [comment]
Yeah very odd situation. Well I'm glad it now works, sad I can't use that mod but I'd rather play the pack either way :D
